### PR TITLE
5.4.6-rc2 history additions

### PR DESCRIPTION
**Pull request by joshmoore**
Add changes between 5.4.6-rc1 and 5.4.6-rc2. Dropped the "rc1" with the expectation that the final release will similarly drop "rc2".

### CODE REVIEW

File: history.txt [hunk 1]
```diff
@@ -16,12 +16,16 @@ of your OMERO installations.
 Further improvements include:
 
 -  enabled big image support in ImageJ/Fiji
--  fixed the handling of script params in OMERO.web
--  fixed bug in the History tab of OMERO.web
+-  reduced the number of threads used by OMERO.web
+-  fixed bugs in OMERO.web including:
+   - History tab
+   - handling of script params
+   - pagination calulcations
```
Comment (jburel) [inline history.txt]: typo

**Comment by mtbc** [comment]
Maybe also mention deprecating `Repository::pixels` in Blitz API and the `TinyImportFixture` Java class.

**Comment by jburel** [comment]
* Added option to set custom physical pixel size using CLI

**Comment by joshmoore** [comment]
Merging with the current set of suggestions. We can have another round of modifications for GA.

**Pull request merged**